src/components/overlays/Sheet/SwipeableSheet/SwipeableSheet.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useCallback, useMemo, useEffect } from 'react'
+import { Animated, LayoutRectangle, ViewProps, PanResponder } from 'react-native'
+import { useStyles, useTheme } from '../../../../theme'
+
+export interface SwipeableSheetProps {
+  open: boolean
+  children: React.ReactNode
+  onHidden: () => void
+}
+
+const hiddenTranslate = 9999999
+
+const springAnimationConfig: Partial<Animated.SpringAnimationConfig> = {
+  velocity: 4,
+  tension: 8,
+  friction: 8,
+  overshootClamping: true,
+  useNativeDriver: true,
+  // Customize rest thresholds so animation ends more swiftly
+  restSpeedThreshold: 1,
+  restDisplacementThreshold: 0.1,
```

Comment:
We can stick with the default values for a snappier feel. Ended up with this:

```
  overshootClamping: true,
  useNativeDriver: true,
  // Customize rest thresholds so animation ends more swiftly
  restSpeedThreshold: 1,
  restDisplacementThreshold: 0.1,
}
```

Reply:
Changed :)